pom.xml

```diff
@@ -764,6 +771,22 @@
             <artifactId>commons-math3</artifactId>
             <version>3.6.1</version>
         </dependency>
+        <dependency>
+            <groupId>javax.measure</groupId>
```

Comment:
What is this dependency for? Please add a brief XML comment saying what it does.

Reply:
Sure !